index.php

```diff
@@ -0,0 +1,38 @@
+<?php
+
+
+$db = new PDO('mysql:host=db; dbname=ukgamesfunded', 'root', 'password');
+
+$db->setAttribute(PDO::ATTR_DEFAULT_FETCH_MODE, PDO::FETCH_ASSOC);
+
+$query = $db->prepare("SELECT `title`, `year` , `theme` , `genre` , `rating`  FROM `game`");
+
+$query->execute();
+
+$result = $query->fetchAll();
+
+echo '<h2>UK Games Funded</h2>';
+
+?>
+<table>
+    <tr>
+        <th>Title</th>
+        <th>Year Funded</th>
+        <th>Theme</th>
+        <th>Genre</th>
+        <th>Add Rating</th>
+    </tr>
+<?php
+foreach ($result as $game) {
+    echo '<tr>';
+    echo '<td>' . $game['title'] . '</td>';
+    echo '<td>' . $game['year'] . '</td>';
+    echo '<td>' . $game['theme'] . '</td>';
+    echo '<td>' . $game['genre'] . '</td>';
+    echo '<td>' . $game['rating'] . '</td>';
+    echo '</tr>';
+}
```

Comment:
And finally, once you've figured out the other two functions, I would suggest putting this into a function as well! One that can be given the list of games, and will return all of the HTML as a string needed to display them in your table.

This has another benefit in that this particular function would be unit-testable, giving you a chance to do some more unit testing practice!